iron-request.html

```diff
@@ -206,6 +206,9 @@
           loaded: progress.loaded,
           total: progress.total
         });
+
+        // Webcomponents v1 spec does not fire events when not connected
+        this.fire('iron-request-progress-changed', { value: this.progress });
```

Comment:
No, this will not happen. It will fire twice if i was firing `progress-changed`, but i have a unique event name. Will update comment accordingly